yellowbox/utils.py

```diff
@@ -31,3 +32,8 @@ def get_free_port():
         s.bind(('', 0))
         s.setsockopt(SOL_SOCKET, SO_REUSEADDR, 1)
         return s.getsockname()[1]
+
+if platform.system() == "Linux":
+    _docker_host_name = '172.17.0.1'
```

Comment:
make it public

Reply:
We don't want external people to use it, do we?
If we do I'll make it public

Reply:
1. maybe we do, if users want to create non-container services, no need for them to reinvent the wheel.
2. I meant only to remove the underscore for the name as we import it in other yellowbox modules.

Reply:
Removing the underscore means it public for module users as well. It's quite common in packages to import underscores internally.

If we want users to be able to use it, sure. We just need to think of a better name, document it, and make sure it works on most operating systems (which i believe it does).